plugins/woocommerce-admin/client/marketplace/components/notice/notice.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { useState } from '@wordpress/element';
+import { Icon, check, closeSmall, info, percent } from '@wordpress/icons'; // See: https://wordpress.github.io/gutenberg/?path=/docs/icons-icon--docs
+
+/**
+ * Internal dependencies
+ */
+import './notice.scss';
+import sanitizeHTML from '../../../lib/sanitize-html';
+
+export interface NoticeProps {
+	id: string;
+	children?: JSX.Element;
```

Comment:
Changing the type to `React.ReactNode` would allow passing multiple children without wrapping them in a fragment:

```suggestion
	children?: React.ReactNode;
```